lib/engine/game/g_1817_de/game.rb

```diff
@@ -0,0 +1,695 @@
+# frozen_string_literal: true
+
+# rubocop:disable Layout/LineLength
+
+require_relative '../g_1817/game'
+require_relative 'meta'
+
+module Engine
+  module Game
+    module G1817DE
+      class Game < G1817::Game
+        include_meta(G1817DE::Meta)
+
+        CURRENCY_FORMAT_STR = '%d ℳ'
+
+        BANK_CASH = 99_999
+
+        CERT_LIMIT = { 2 => 21, 3 => 16, 4 => 13, 5 => 11, 6 => 9 }.freeze
+
+        STARTING_CASH = { 2 => 420, 3 => 315, 4 => 252, 5 => 210, 6 => 180 }.freeze
+
+        CAPITALIZATION = :incremental
+
+        MUST_SELL_IN_BLOCKS = false
+
+        TILES = {
```

Comment:
can you split the map stuff and entity stuff into files names map and entities (see how 18Ireland did it).
This makes it a bit easier to navigate the code

Reply:
Okay, I split these out now